server/src/main/java/org/elasticsearch/action/admin/cluster/node/capabilities/TransportNodesCapabilitiesAction.java

```diff
@@ -159,7 +164,9 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeString(path);
             out.writeCollection(parameters, StreamOutput::writeString);
             out.writeCollection(capabilities, StreamOutput::writeString);
-            out.writeVInt(restApiVersion.major);
+            // Fixme: lies! all lies!
+            out.writeVInt(8);
```

Comment:
We need to decide what to do when asked the capabilities of a mixed v8/v9 cluster and a v9 rest api - probably to check the transport versions/available rest API versions beforehand, and answer the check before querying any of the nodes.

Reply:
@thecoop - are you OK with how it is for this PR and address the proper fix later ?

Reply:
Yes, this can be merged as-is - the `@UpdateForV9` will prompt us to fix it later